### PR TITLE
warn log if empty response

### PR DESCRIPTION
**Pull request by rassmate**
Logging information when a response is empty

### CODE REVIEW

File: client/src/main/java/se/fortnox/reactivewizard/client/HttpClient.java [hunk 1]
```diff
@@ -188,6 +196,15 @@ private void logFailedRequest(RequestBuilder fullReq, Throwable throwable) {
             .flatMap(str -> deserialize(method, str));
     }
 
+    private Observable<? extends ByteBuf> logEmptyResponse(RequestBuilder request, HttpClientResponse<ByteBuf> response) {
+        return defer(() -> {
+
```
Comment (jepp3) [inline client/src/main/java/se/fortnox/reactivewizard/client/HttpClient.java]: can we use the loggers built in supplant function instead?

**Comment by splitfeed** [comment]
Wont this result in a lot of logs for endpoints that simply return no body, like 204:s or just empty 200:s?

**Comment by JonasHallFnx** [comment]
> Wont this result in a lot of logs for endpoints that simply return no body, like 204:s or just empty 200:s?

I agree, we should check that expected returntype is not Observable<Void> I think.

**Comment by JonasHallFnx** [comment]
Have you seen this? https://github.com/ReactiveX/RxNetty/issues/511

File: client/src/main/java/se/fortnox/reactivewizard/client/HttpClient.java (hunk 1 above)
Comment (rassmate) [reply client/src/main/java/se/fortnox/reactivewizard/client/HttpClient.java]: Since  I need to return an observable here I think not

**Pull request merged**